Automattic/newspack-theme
# fix: switch menu dropdowns to use primary color to improve contrast

### All Submissions:


### Changes proposed in this Pull Request:

The menu dropdowns originally used a darker version of the primary colour as a background on hover. This works well with the default blue for contrast (since the text would be white against the primary colour and the darker version of the primary colour), but in some cases, there isn't sufficient contrast. You can pick a primary colour that's light enough that the theme would pair it with black text, but the darker version of the primary colour as a background would make that difficult to read. 

This PR switches the dropdowns to just use the primary colour. 

Closes #1087.

### How to test the changes in this Pull Request:

1. Set up a primary menu with dropdowns.
2. Set a primary colour that's dark-ish, but would use black as a text colour -- for example, `#f25800`.
3. View the dropdown menu:

![image](https://user-images.githubusercontent.com/177561/94494470-1eec5080-01a4-11eb-93ce-dc2a3d3fc596.png)

4. Apply the PR and run `npm run build`.
5. Check the dropdown menu again; the background will be a bit lighter, and have better contrast:

![image](https://user-images.githubusercontent.com/177561/94494571-5e1aa180-01a4-11eb-97d9-57611fdbf4e2.png)

### Other information:
